### ISSUE TITLE
Official API docs may be outdated

### ISSUE THREAD

**Issue by bingzhangdai** [microsoft/GraphEngine#122]
I roughly go through the [API reference](https://www.graphengine.io/APIRefs/html/N_Trinity.htm) and find that it might be outdated. For instance, [```NewCellID```](https://www.graphengine.io/APIRefs/html/M_Trinity_Core_Lib_CellIDFactory_NewCellID.htm) method is obsolete and ``` NewCellId ``` is recommended according to the source code (Although they have no difference currently). [```SetPartitionMethod```](https://www.graphengine.io/APIRefs/html/M_Trinity_TrinityConfig_SetPartitionMethod.htm) actually does not belong to ```TrinityConfig``` class. You can find it in ```MemoryCloud``` instead.

Surely, adding more functionality is absolutely necessary. A well-written documentation can attract more developers and can, in return, make Graph Engine better. :)

**Comment by yatli** [comment]
@bingzhangdai the API surface has been changing lately, and I'm having a lot of stuff written about the updates, including an XSD file for trinity config :) I will work with Bin to update the site once we stablize everything!

**Comment by bingzhangdai** [comment]
Looking forward to new features. 👍

**Comment by yatli** [comment]
@bingzhangdai update: see #120, it will be settled down soon! and there's a lot to be written about!

**Comment by yatli** [comment]
linking #66

**Comment by TaviTruman** [comment]
@yatli Hey Yatao my workload is starting to slow down and I have more time to spend on GraphEngine; I am already doing some of our writing around API usage.

**Comment by yatli** [comment]
@TaviTruman  that's good to know!

**Comment by TonyLo1** [comment]
What is the recommended approach to build API docs? The link from GraphEngine.io no longer functions (404 error)

**Comment by TaviTruman** [comment]
@TonyLo1 I'm not sure what the approach is to build the docs as I've not witnessed any particular method other than members of the Trinity and Graph Engine teams updating the docs. The docs aren't part of the current build release process; there may be another method and a process whereby the docs are updated and published but I don't have access. As original Graph Engine team members are no longer maintaining this repro, I have volunteered to get the site updated because I'm actively building Azure Service Fabric microservices as well as a whole suite of tooling atop the GE; I have a small team of engineers that maintain our forked repo, and we have made numerous additions to the code and to the docs. As I'm not able to gain access to the production repro we can't update the CI/CD workflow. We have been working on our own CI/CD build and deployment workflow, and look to have it ready to go come January 2021. It does seem that the Graph Engine team has pulled down the API Docs content; I will make a direct inquiry.
